utils/build-script-impl

```diff
@@ -1968,6 +1980,17 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCMAKE_SHARED_LINKER_FLAGS:STRING="-fuse-ld=gold"
                 )
             fi
+        elif [[ "${USE_LLD_LINKER}" ]]; then
```

Comment:
Can you split the LLD change into a separate PR. That should be able to stand on its own.

Reply:
Yeah, that is the easiest to pull out.